hack/loadtest/docker-compose.yml

```
@@ -2,7 +2,7 @@ version: "3.9"
 services:
   cerbos:
     container_name: cerbos
-    image: ghcr.io/cerbos/cerbos:0.13.0
+    image: ghcr.io/cerbos/cerbos:dev
```

Perhaps we could make this easier to override?

```suggestion
    image: ghcr.io/cerbos/cerbos:${CERBOS_VERSION:-dev}
```